app/lib/features/chat/widgets/image_message_builder.dart

```diff
@@ -46,48 +47,64 @@ class _ImageMessageBuilderState extends State<ImageMessageBuilder> {
     }
   }
 
+  void showFullScreenImage(Uint8List data) {
+    showAdaptiveDialog(
+      context: context,
+      barrierDismissible: false,
+      useRootNavigator: false,
+      builder: (ctx) => ImageDialog(
+        title: 'Image',
```

Comment:
shouldn't the image-message have a name as well?
```suggestion
        title: widget.message.name(),
```